src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx

```diff
@@ -0,0 +1,208 @@
+          metadata.startingProtectionYear,
+        ]
+      : [];
+
+    return (
+      (isTreeProject && treeProjectConditions.some(Boolean)) ||
+      (isConservationProject && conservationProjectConditions.some(Boolean)) ||
+      generalConditions.some(Boolean)
+    );
+  }, [metadata, isTreeProject, isConservationProject]);
+
+  const shouldRenderAdditionalInfo = useMemo(() => {
+    const {
+      mainChallenge,
+      siteOwnerName,
+      longTermPlan,
+      acquisitionYear,
+      motivation,
+      mainInterventions,
+    } = metadata;
+    const generalConditions = [
+      mainChallenge,
+      siteOwnerName,
+      longTermPlan,
+      acquisitionYear,
+      motivation,
+      mainInterventions && mainInterventions?.length > 0,
+    ];
+    const treeProjectConditions = isTreeProject
+      ? [
+          metadata.siteOwnerType && metadata.siteOwnerType?.length > 0,
+          metadata.degradationCause,
+        ]
+      : [];
+    const conservationProjectConditions = isConservationProject
+      ? [
+          metadata.landOwnershipType && metadata.landOwnershipType?.length > 0,
+          metadata.actions,
+          metadata.socialBenefits,
+          metadata.ecologicalBenefits,
+          metadata.coBenefits,
+          metadata.benefits,
+          metadata.ownershipType,
+        ]
+      : [];
+
+    return (
+      generalConditions.some(Boolean) ||
+      treeProjectConditions.some(Boolean) ||
+      conservationProjectConditions.some(Boolean)
+    );
+  }, [metadata]);
+
+  const location = useMemo(() => {
+    if (!tpo.address) return '';
+    const { address, zipCode, city, country } = tpo.address;
+    return [
+      address,
+      zipCode,
+      city,
+      country
+        ? tCountry(tpo.address.country.toLowerCase() as Lowercase<CountryCode>)
+        : '',
+    ]
+      .filter(Boolean)
+      .join(',');
+  }, [tpo]);
+
+  const shouldRenderProjectDownloads = useMemo(() => {
+    return certificates.length > 0 || expenses.length > 0;
+  }, [certificates, expenses]);
+  const handleMap = () => setSelectedMode?.('map');
+
+  return (
+    <section className={styles.projectInfoSection}>
+      {reviews?.length > 0 && <ProjectReview reviews={reviews} />}
+      {description && <AboutProject description={description} />}
+      {videoUrl && <VideoPlayer videoUrl={videoUrl} />}
+      {images.length > 0 && (
+        <ImageSlider images={images} type={'project'} isMobile={isMobile} />
+      )}
+      {isMobile && <MapPreview handleMap={handleMap} />}
+      {shouldRenderKeyInfo && (
+        <KeyInfo
+          abandonment={isTreeProject ? metadata.yearAbandoned : null}
+          interventionStarted={
+            isTreeProject
+              ? metadata.firstTreePlanted
+              : metadata.startingProtectionYear
+          }
+          interventionSeasons={
+            isTreeProject ? metadata.plantingSeasons : metadata.activitySeasons
+          }
```

Comment:
I think that combining 2 different data e.g. firstTreePlanted and startingProtectionYear makes the code more complicated. We need to add conditions while displaying this to check the project type, instead of simply checking that the data is not null. Consider passing in each separate data point separately.